index.js

```diff
@@ -1449,11 +1449,11 @@
     impl: curry2 (Z.contramap)
   };
 
-  //. ### Combinator
+  //. ### Combinator: A function or definition with no free variables.
```

Comment:
This is the current implementation of `T`:

```javascript
function T(x) {
  return function(f) {
    return f (x);
  };
}
```

Arrow functions would make the definition clearer:

```javascript
const T = x => f => f (x);
```

This could be written `Txf = fx` which is equivalent to `Txy = yx`. I believe `S.T` really is the thrush combinator.